src/import_record.zig

```diff
@@ -161,6 +161,8 @@ pub const ImportRecord = struct {
     wrap_with_to_esm: bool = false,
     wrap_with_to_commonjs: bool = false,
 
+    tailwind: bool = false,
```

Comment:
Is this boolean necessary? If so, can we add it to the list of tags instead of adding an extra 1 byte to this struct? Eventually, we should move all the booleans here to a bitset